### ISSUE TITLE
Create unit test for be_refresh

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1944]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1940
- Created at 2013-05-23 10:58:05 by [pbrezina](https://pagure.io/user/pbrezina)
- Closed at 2020-03-24 14:18:36 as wontfix
- Assigned to [mzidek](https://pagure.io/user/mzidek)
---

Create unit test for be_refresh located at src/providers/dp_refresh.c.

### Comments

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2013-05-23 15:40:43](https://pagure.io/SSSD/sssd/issue/1940#238604)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.11 beta
type: defect => task

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2013-05-23 15:40:53](https://pagure.io/SSSD/sssd/issue/1940#238605)**

Fields changed

rhbz:  => 0

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2013-06-28 23:55:07](https://pagure.io/SSSD/sssd/issue/1940#238606)**

Moving open tickets from 1.11 beta to 1.11 beta2

milestone: SSSD 1.11 beta => SSSD 1.11 beta 2

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2013-07-24 14:53:55](https://pagure.io/SSSD/sssd/issue/1940#238607)**

Fields changed

changelog:  => 
milestone: SSSD 1.11 beta 2 => Interim Bucket

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2013-07-30 12:54:43](https://pagure.io/SSSD/sssd/issue/1940#238608)**

Fields changed

milestone: Interim Bucket => SSSD 1.12 beta

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-02-19 11:53:56](https://pagure.io/SSSD/sssd/issue/1940#238609)**

Fields changed

blocking: 1923 => #1923

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-05-30 15:53:35](https://pagure.io/SSSD/sssd/issue/1940#238610)**

Fields changed

milestone: SSSD 1.12 beta => SSSD 1.12.1

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-09-08 20:09:05](https://pagure.io/SSSD/sssd/issue/1940#238611)**

Mass-moving all tickets that didn't make 1.12.1 into 1.12.2

milestone: SSSD 1.12.1 => SSSD 1.12.2

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-09-30 19:06:19](https://pagure.io/SSSD/sssd/issue/1940#238612)**

We need to do a release as requested by downstream. Moving tickets that are not fixed already or very close to acking to 1.12.3

milestone: SSSD 1.12.2 => SSSD 1.12.3

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-11-13 19:17:46](https://pagure.io/SSSD/sssd/issue/1940#238613)**

Fields changed

mark:  => 0
owner: somebody => mzidek

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2014-11-19 17:37:42](https://pagure.io/SSSD/sssd/issue/1940#238614)**

It makes sense to write the unit tests now instead of moving them to a later release again, but the tickets in 1.12.3 should be more focused at this point. Moving to 1.12.4

milestone: SSSD 1.12.3 => SSSD 1.12.4

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2015-02-12 20:05:23](https://pagure.io/SSSD/sssd/issue/1940#238615)**

Fields changed

milestone: SSSD 1.12.4 => SSSD Deferred

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-02-24 14:58:34](https://pagure.io/SSSD/sssd/issue/1940#238616)**

**Metadata Update from @pbrezina**:
- Issue assigned to mzidek
- Issue marked as blocked by: #1923
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:18:35](https://pagure.io/SSSD/sssd/issue/1940#636902)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:18:36](https://pagure.io/SSSD/sssd/issue/1940#636903)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
